landingai/pipeline/image_source.py

```diff
@@ -337,3 +338,37 @@ def __next__(self) -> "FrameSet":
 
     class Config:
         arbitrary_types_allowed = True
+
+
+class Webcam(NetworkedCamera):
+    """
+    The Webcam class can connect to a local webcam in order to grab frames.
+
+    This leverages the NetworkedCamera implementations, with the constructor
+    receiving the webcam ID to be sent to OpenCV (instead of a stream URL).
+    """
+
+    def __init__(
+        self,
+        webcam_source: int = 0,
+        motion_detection_threshold: int = 0,
+        capture_interval: Optional[float] = None,
+        fps: Optional[int] = None,
+    ) -> None:
+        super().__init__(
+            webcam_source,
+            motion_detection_threshold=motion_detection_threshold,
+            capture_interval=capture_interval,
+            fps=fps,
+        )
+
+
+class Screenshot(ImageSourceBase):
+    """Take a screenshot from the screen as an image source."""
```

Comment:
an explanation about when the screenshot is taken is helpful for novice developers.
E.g. it only takes a screen shot when you iterate over this object. For example, every iteration in a for loop takes a new screenshot.

Reply:
LGTM, thanks!